src/v/cloud_roles/tests/BUILD

```diff
@@ -30,3 +30,32 @@ redpanda_cc_btest(
         "@boost//:test",
     ],
 )
+
+redpanda_cc_btest(
+    name = "cloud_roles_tests",
+    timeout = "short",
+    srcs = [
+        "categorization_tests.cc",
+        "credential_tests.cc",
+        "env_override_tests.cc",
+        "fetch_credentials_tests.cc",
+        "role_client_tests.cc",
+        "test_definitions.h",
+    ],
+    cpu = 1,
+    tags = ["exclusive"],
```

Comment:
remove the exclusive tag? https://github.com/redpanda-data/redpanda/commit/b5b877e291159617e281ce210360afb6cb4d131b